src/components/contacts-list/ContactsList.js

```diff
@@ -36,13 +43,13 @@ class ContactsList extends React.Component {
 }
 
 ContactsList.defaultProps = {
-  contactsList: [],
+  contacts: {},
   title: '',
-  contactMethods: {}
+  areFavourites: false
 };
 
 ContactsList.propTypes = {
-  contactsList: PropTypes.arrayOf(PropTypes.exact({
+  contacts: PropTypes.objectOf(PropTypes.exact({
```

Comment:
Isn't it possible to have this come as an array to this component? Maybe make the required conversions before sending this as props.